Reiniciar do trocar tema está dando erro

Quando se tenta trocar o tema, o mesmo pede para reiniciar, mas acaba gerando um erro.

![image](https://user-images.githubusercontent.com/8836540/30883651-387a3b20-a2e3-11e7-8f81-a23da4fd1123.png)


``` br.univali.ps.nucleo.ExcecaoAplicacao: Uma das bibliotecas ou classes necess￡rias para o funcionamento do Portugol Studio n￣o foi encontrada.
O Portugol Studio ser￡ enecerrado.
	at br.univali.ps.nucleo.TratadorExcecoes.uncaughtException(TratadorExcecoes.java:159)
	at java.lang.ThreadGroup.uncaughtException(ThreadGroup.java:1057)
	at java.lang.ThreadGroup.uncaughtException(ThreadGroup.java:1052)
	at java.awt.EventDispatchThread.processException(EventDispatchThread.java:223)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:215)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:109)
	at java.awt.WaitDispatchSupport$2.run(WaitDispatchSupport.java:184)
	at java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:229)
	at java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:227)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.awt.WaitDispatchSupport.enter(WaitDispatchSupport.java:227)
	at java.awt.Dialog.show(Dialog.java:1084)
	at java.awt.Component.show(Component.java:1671)
	at java.awt.Component.setVisible(Component.java:1623)
	at java.awt.Window.setVisible(Window.java:1014)
	at java.awt.Dialog.setVisible(Dialog.java:1005)
	at br.univali.ps.ui.abas.AbaInicial$14.actionPerformed(AbaInicial.java:448)
	at br.univali.ps.ui.abas.AbaInicial$12.mouseClicked(AbaInicial.java:363)
	at java.awt.Component.processMouseEvent(Component.java:6536)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
	at java.awt.Component.processEvent(Component.java:6298)
	at java.awt.Container.processEvent(Container.java:2236)
	at java.awt.Component.dispatchEventImpl(Component.java:4889)
	at java.awt.Container.dispatchEventImpl(Container.java:2294)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4888)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4534)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4466)
	at java.awt.Container.dispatchEventImpl(Container.java:2280)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:90)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.awt.EventQueue$4.run(EventQueue.java:729)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: java.lang.NoClassDefFoundError: java/lang/management/ManagementFactory
	at br.univali.ps.nucleo.Configuracoes.restartApplication(Configuracoes.java:337)
	at br.univali.ps.nucleo.Configuracoes.TrocarTema(Configuracoes.java:308)
	at br.univali.ps.ui.telas.TelaDicas.webButton3ActionPerformed(TelaDicas.java:284)
	at br.univali.ps.ui.telas.TelaDicas.access$500(TelaDicas.java:38)
	at br.univali.ps.ui.telas.TelaDicas$6.actionPerformed(TelaDicas.java:256)
	at javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:2022)
	at javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2348)
	at javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:402)
	at javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:259)
	at javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:252)
	at java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:289)
	at java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:289)
	at java.awt.Component.processMouseEvent(Component.java:6533)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
	at java.awt.Component.processEvent(Component.java:6298)
	at java.awt.Container.processEvent(Container.java:2236)
	at java.awt.Component.dispatchEventImpl(Component.java:4889)
	at java.awt.Container.dispatchEventImpl(Container.java:2294)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4888)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4525)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4466)
	at java.awt.Container.dispatchEventImpl(Container.java:2280)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:90)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.awt.EventQueue$4.run(EventQueue.java:729)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	... 45 more
Caused by: java.lang.ClassNotFoundException: java.lang.management.ManagementFactory
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:335)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 83 more
```


Erro se encontra aqui:
![image](https://user-images.githubusercontent.com/8836540/30883851-e952cdf4-a2e3-11e7-8621-2e25bf546bdc.png)

--- a/installer/jdk-minify-profile.json
+++ b/installer/jdk-minify-profile.json
@@ -12,7 +12,6 @@
         "jdk/jre/lib/desktop",
         "jdk/jre/lib/ext",
         "jdk/jre/lib/jfr",
-        "jdk/jre/lib/management",
         "jdk/jre/lib/oblique-fonts",
 
         "jdk/jre/plugin",
@@ -24,7 +23,6 @@
         "jdk/jre/lib/javaws.jar",
         "jdk/jre/lib/jfr.jar",
         "jdk/jre/lib/jfxswt.jar",
-        "jdk/jre/lib/management-agent.jar",
         "jdk/jre/lib/plugin.jar",
 
         "jdk/jre/THIRDPARTYLICENSEREADME-JAVAFX.txt",
@@ -147,7 +145,6 @@
         "jdk/jre/bin/jfxmedia.dll",
         "jdk/jre/bin/jfxwebkit.dll",
         "jdk/jre/bin/jfr.dll",
-        "jdk/jre/bin/management.dll",
         "jdk/jre/bin/prism_common.dll",
         "jdk/jre/bin/prism_d3d.dll",
         "jdk/jre/bin/prism_es2.dll",
@@ -306,7 +303,6 @@
         "jdk/jre/lib/i386/libjfxmedia.so",
         "jdk/jre/lib/i386/libjfxwebkit.so",
         "jdk/jre/lib/i386/libjsig.so",
-        "jdk/jre/lib/i386/libmanagement.so",
         "jdk/jre/lib/i386/libprism_common.so",
         "jdk/jre/lib/i386/libprism_es2.so",
         "jdk/jre/lib/i386/libprism_sw.so"
@@ -382,7 +378,6 @@
         "jdk/jre/lib/amd64/libjfxmedia.so",
         "jdk/jre/lib/amd64/libjfxwebkit.so",
         "jdk/jre/lib/amd64/libjsig.so",
-        "jdk/jre/lib/amd64/libmanagement.so",
         "jdk/jre/lib/amd64/libprism_common.so",
         "jdk/jre/lib/amd64/libprism_es2.so",
         "jdk/jre/lib/amd64/libprism_sw.so"

